relayer/relayer.go

```diff
@@ -3,6 +3,8 @@ package relayer
 import (
 	"context"
 	"errors"
+	"github.com/NethermindEth/near-sffl/core"
```

Comment:
These imports could be separated into std, external and internal

Reply:
Done